src/test/java/com/facebook/openwifirrm/optimizers/tpc/RandomTxPowerInitializerTest.java

```diff
@@ -63,16 +66,32 @@ private static DataModel createModel() {
 		return dataModel;
 	}
 
+	/**
+	 * Return the first value provided by Random, for each key seed
+	 */
+	private static Map<Integer, Integer> mapRngSeedToFirstValue() {
+		Map<Integer, Integer> map = new HashMap<>();
+		map.put(0, 2);
+		map.put(10, 9);
+		map.put(100, 10);
```

Comment:
nit: can you just put this into the test method? it is more confusing to me that this function exists at all